agent/Cargo.toml

```diff
@@ -13,7 +13,7 @@ blake2 = "0.8.0"
 chrono = "0.4.10"
 cfg-if = "0.1"
 env_logger = "0.6.1"
-failure = "0.1.5"
+anyhow = "1.0.38"
```

Comment:
nit: should we fix the ordering?

Reply:
good catch, @jiria. we should. would you be willing to put in the fix?